src/Marten/Internal/OpenTelemetry/MartenTracing.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Marten.Internal.OpenTelemetry;
+
+internal static class MartenTracing
+{
+    // See https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/messaging/ for more information
+    public static string MartenTenantId = "MartenTentantId";
+    public static string MartenCorelationId = "MartenCorelationId";
```

Comment:
Can these be `const` are `static readonly`?
When they should be setable, then make them a property.